code/game/objects/items/weapons/grenades/clusterbuster.dm

```diff
@@ -120,3 +120,7 @@
 /obj/item/weapon/grenade/clusterbuster/spawner_spesscarp
 	name = "Invasion of the Space Carps"
 	payload = /obj/item/weapon/grenade/spawnergrenade/spesscarp
+
+/obj/item/weapon/grenade/clusterbuster/soap
+	name = "Slipocalypse"
+	payload = /obj/item/weapon/soap/syndie
```

Comment:
On our server we just did a snowflake check that if it was a grenade, then it would set the ready var to 1 and prime it, otherwise, that var and prime() would not be called.